### PR TITLE
Add page: Media

### PR DESCRIPTION
**Pull request by straight-shoota**
Goes on top of https://github.com/crystal-lang/crystal-website/pull/480

### CODE REVIEW

File: _pages/media.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Thanks for showing interest in Crystal! As the language and the community
+continues to grow we think it is important to have basic rules of thumb about
+how and for what purposes you can use the Crystal brand.
+
+We’ve put together some assets and guidelines to help you determine the best way
+to use our brand and logos.
+
+### Logo
+
+<div class="cards-list">
```
Comment (beta-ziliani) [inline _pages/media.md:20]: Ensure people follow the guidelines, something of the form:

```suggestion

> **NOTE** Important
> Make sure to follow the [guidelines](???) described below when using any of this material.

<div class="cards-list">
```

**Review by beta-ziliani — CHANGES_REQUESTED** [review]

File: _pages/media.md (hunk 1 above)
Comment (straight-shoota) [reply _pages/media.md:20]:
```suggestion

> **NOTE** Important
> Make sure to follow the [guidelines](#guidelines) described below when using any of this material.

<div class="cards-list">
```

Comment (beta-ziliani) [reply _pages/media.md:20]: The callout is coming weird (everything in one line), despite of following the rules described in the `style-guide`
image

Comment (beta-ziliani) [reply _pages/media.md:20]: it's missing the `:`, that might be it?

**Comment by beta-ziliani** [comment]
still no 😢

**Review by beta-ziliani — APPROVED** [review]
finally! 🚀

**Pull request merged**